gradle.properties

```diff
@@ -1,13 +1,13 @@
 
 # Terracotta third parties
 offheapVersion = 2.4.0
-statisticVersion = 2.0.1
+statisticVersion = 2.0-SNAPSHOT
```

Comment:
Need new release after https://github.com/Terracotta-OSS/statistics/pull/46 is merged